pkg/apis/operations/v1alpha1/register.go

```diff
@@ -65,6 +65,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&NodeUpgradeJob{},
 		&NodeUpgradeJobList{},
+		&ImagePrepullJob{},
+		&ImagePrepullJobList{},
```

Comment:
Now we place the ImagePrepullJob under operations api group, another api group `apps.kubeedge.io` should be also fine, and ImagePrepullJob is mainly used for app,  how doyou think? @WillardHu @ZhengXinwei-F @cl2017

Reply:
I considered from the architecture level before. The APIs that use the upgrade framework are placed under `operation.kubeedge.io`. From the functional point of view, it is more appropriate to put them in the `apps.kubeedge.io`.